### PR TITLE
Add Nijmegen's student association Thalia as issuer and verifier

### PR DESCRIPTION
**Pull request by koen92**
This pull request adds xml files to support verification of credentials used by the IRMA pilot within Nijmegen's student association Thalia.

### CODE REVIEW

File: Thalia/gp.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<GroupParameters xmlns="http://www.zurich.ibm.com/security/idemix" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.zurich.ibm.com/security/idemix GroupParameters.xsd">
+  <References>
+    <SystemParameters>http://thalia.nu</SystemParameters>
```
Comment (puiterwijk) [inline Thalia/gp.xml]: This should be a url to the gp.xml file, prefixed by the Thalia/description.xml baseURL.
It doesn't need to actually exist, but needs to be "valid" in that sense.

File: Thalia/ipk.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<IssuerPublicKey xmlns="http://www.zurich.ibm.com/security/idemix" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.zurich.ibm.com/security/idemix IssuerPublicKey.xsd">
+  <References>
+    <GroupParameters>http://www.irmacard.org/credentials/phase1/Thalia/gp.xml</GroupParameters>
```
Comment (puiterwijk) [inline Thalia/ipk.xml]: Same comment as gp.xml.

**Comment by wouterl** [comment]
General comment: we have 3 branches to keep up to date: master, pilot and demo. Please structure your pull request so I can deal with that:

 1. Drop the public key, and suggest a merge for this to master
 2. Commit the public key seperately on demo

I'll make sure to add this structure to demo (with default key) and pilot.

**Comment by wouterl** [comment]
Thanks for the pull request, please fix the indicated issues.

**Comment by koen92** [comment]
You wrote:
> Commit the public key seperately on demo

Are you sure that I have to commit the public key on the demo branch? This Thalia pilot is part (and a subset of) the IRMA Kerckhoff's pilot, so I think that the public key belongs to the pilot branch instead of the demo branch?

**Comment by wouterl** [comment]
On Mon, Jan 19, 2015 at 06:19:46AM -0800, koen92 wrote:
> You wrote:
> 
>     Commit the public key seperately on demo
> 
> Are you sure that I have to commit the public key on the demo branch? This
> Thalia pilot is part (and a subset of) the IRMA Kerckhoff's pilot, so I think
> that the public key belongs to the pilot branch instead of the demo branch?

You are correct. No keys in master, and the public key you generated in
the pilot branch. Thanks for checking.

**Pull request closed without merging**